WeScan/Scan/CaptureSessionManager.swift

```diff
@@ -167,20 +163,16 @@ final class CaptureSessionManager: NSObject, AVCaptureVideoDataOutputSampleBuffe
         var motion: CMMotionManager!
         motion = CMMotionManager()
         
+        CaptureSession.current.editImageOrientation = .up
```

Comment:
Can we make this whole method an extension on `CaptureSession`, like `CaptureSession+Orientation.swift`, where the implementation would be:

```swift
CaptureSession.current.updateImageOrientation()
```

Separation of concern and keeping our code focused 👌

Reply:
👍 Will work on this

Reply:
Done in 6fcdb19